python/lsst/validate/drp/plot.py

```diff
@@ -118,8 +118,9 @@ def plotAstrometryErrorModel(dataset, astromModel, outputPrefix=''):
 
     ax[0].hist(dist, bins=100, color=color['all'],
```

Comment:
Are we confident that dist will always be non-zero length, or is the statement that if dist is zero length we better except because we have bigger problems?

Reply:
I hadn't really thought about the general case of someone passing in an empty dataset for plotting.  Maybe it should raise a ValueError as soon as numMatched is calculated?

Reply:
Or maybe you expect an empty plot? I'll merge this as is and worry about the other problem later.